modules/xmpp/JingleSessionPC.js

```diff
@@ -412,8 +412,9 @@ JingleSessionPC.prototype.replaceTransport = function (jingleOfferElem,
             this.setOfferCycle(
                 originalOffer,
                 () => {
-                    // Set local description OK, now localSDP up to date
-                    this.sendTransportAccept(this.localSDP, success, failure);
+                    const localSDP
+                        = new SDP(this.peerconnection.localDescription.sdp);
+                    this.sendTransportAccept(localSDP, success, failure);
```

Comment:
so this was broken because `this.localSDP` was removed, right?  i think i may have done that so maybe i caused this bug.  if so then lgtm.